tests/TestGame.ts

```diff
@@ -18,13 +20,9 @@ export function newTestGame(count: number, customOptions?: Partial<GameOptions>,
   const options: GameOptions | undefined = customOptions === undefined ?
     undefined :
     setCustomGameOptions(customOptions);
-  return Game.newInstance(`game-id${idSuffix}`, players, players[0], options);
-}
-
-export function getTestPlayer(game: Game, idx: number): TestPlayer {
-  return game.getPlayers()[idx] as TestPlayer;
-}
+  const game = Game.newInstance(`game-id${idSuffix}`, players, players[0], options);
```

Comment:
```suggestion
  const game: TestGame = Game.newInstance(`game-id${idSuffix}`, players, players[0], options);
  game.testPlayers = players;
  return game;
```

Save an _any_ and be proud!

Reply:
Unfortunately this doesn't work.
![image](https://user-images.githubusercontent.com/413481/189782919-41bdc8de-ebdb-4f32-b695-2a63f19bdb54.png)

Reply:
When I make the change locally the build works for me. Is this error seen while building? I suggested the change after being able to build locally.